validate.lic

```diff
@@ -109,12 +109,19 @@ class DRYamlValidator
   def assert_that_crafting_training_spells_are_skills(settings)
     return unless settings.crafting_training_spells
 
-    valid_crafting_training_spell_skills = %w[Augmentation Utility Warding]
+    valid_crafting_training_spells_skills = %w[Augmentation Utility Warding]
 
     settings.crafting_training_spells
             .keys
             .reject { |skill_name| valid_crafting_training_spells_skills.include?(skill_name) }
-            .each { |skill_name| error("Invalid crafting_training_spells: skill name '#{skill_name}'. Valid skill names are '#{valid_crafting_training_spell_skills}'") }
+            .each { |skill_name| error("Invalid crafting_training_spells: skill name '#{skill_name}'. Valid skill names are '#{valid_crafting_training_spells_skills}'") }
+  end
+
+  def assert_that_crafting_training_spells_have_no_cambrinth(settings)
+    return unless settings.crafting_training_spells
+
+    settings.crafting_training_spells.select { |_spell, data| data['cambrinth'] }
+            .each { |_spell, data| warn("#{data['abbrev']} in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.") }
```

Comment:
❓ I think `spell` (the unused first arg) might be the full spell name, which might be better to print than the abbrev. What do you think?

Reply:
That looks better

[validate: WARNING:< Augmentation in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.  >]